enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityMemberStateMachine.java

```diff
@@ -203,10 +204,9 @@ else if ( role.equals( HighAvailabilityModeSwitcher.SLAVE ) )
                 {
                     HighAvailabilityMemberState oldState = state;
                     state = state.slaveIsAvailable( context, instanceId, roleUri );
-                    logger.debug( "Got slaveIsAvailable(" + instanceId + "), " +
-                            "moved to " + state + " from " + oldState );
                     final HighAvailabilityMemberChangeEvent event = new HighAvailabilityMemberChangeEvent( oldState,
                             state, instanceId, roleUri );
+                    logger.debug( "Got slaveIsAvailable(" + instanceId + "), " + "changed " + oldState + " -> " + state );
```

Comment:
They are actually different info. The reason I changed the logging in this class because we have two different ways of print this info in the very same class, such as https://github.com/neo4j/neo4j/blob/master/enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityMemberStateMachine.java#L164. But I find use 'moved to' as key word to search HA state changes is a very good idea. (I use 'Got' before) Then let's use keep 'moved to' and move 'oldState -> state'.